src/main/kotlin/com/bugsnag/android/gradle/BugsnagPlugin.kt

```diff
@@ -366,6 +398,25 @@ class BugsnagPlugin : Plugin<Project> {
         }
     }
 
+    @Suppress("LongParameterList")
+    private fun registerGenerateUnityMappingTask(
+        project: Project,
+        variant: ApkVariant,
+        output: ApkVariantOutput,
+        bugsnag: BugsnagPluginExtension,
+        manifestInfoFileProvider: Provider<RegularFile>
+    ): TaskProvider<out BugsnagGenerateUnitySoMappingTask> {
+        // Create a Bugsnag task to upload Unity mapping file(s)
+        val outputName = taskNameForOutput(output)
+        val taskName = "generateBugsnagUnity${outputName}Mapping"
+        return BugsnagGenerateUnitySoMappingTask.register(project, taskName) {
+            variantOutput = output
+            objDumpPaths.set(bugsnag.objdumpPaths)
+            manifestInfoFile.set(manifestInfoFileProvider)
+            // TODO update me
```

Comment:
This will require additional inputs when the task actually generates mapping files, but for now requires no action